index.html

```diff
@@ -1001,6 +1119,87 @@ <h2>
   top: anchor(--my-anchor-media-query top);
   right: anchor(--my-anchor-media-query right);
 }</code></pre>
+    </section>
+    <section id="anchor-manual" class="demo-item" style="position: relative">
+      <h2>
+        <a href="#manual" aria-hidden="true">🔗</a>
+        Manually apply polyfill
```

Comment:
I wonder if there's a clearer description for the intent of this change. Maybe:

```suggestion
        Polyfill applied to only specified styles
```

Or something along those lines?

Reply:
How about "Manually apply polyfill to specific styles"?